azure-devops/azext_devops/dev/common/exception_handler.py

```diff
@@ -12,7 +12,7 @@
 def azure_devops_exception_handler(ex):
     # we are taking dependency on string here because taking dependency on
     # Azure DevOps Client will increase load time for every command
-    if type(ex).__name__ == 'VstsServiceError':
+    if type(ex).__name__ == 'AzureDevOpsServiceError':
         logger.debug('handling vsts service error')
```

Comment:
are we fixing logging as well in this PR?  'handling vsts service error' => 'handling azure devops service error'

Reply:
No, this one was just functional changes anything that was caught in the tests.

But I will take any non functional changes like this in a follow up Pr.